src/quicklookUI.cs

```diff
@@ -168,5 +168,9 @@ interface QLPreviewingController {
 #endif
 		[Export ("preparePreviewOfSearchableItemWithIdentifier:queryString:completionHandler:")]
 		void PreparePreviewOfSearchableItem (string identifier, string queryString, Action<NSError> ItemLoadingHandler);
+
+		[Mac (10,15, onlyOn64: true)]
+		[Export ("preparePreviewOfFileAtURL:completionHandler:")]
+		void PreparePreviewOfFile (NSUrl url, Action<NSError> ItemLoadingHandler);
```

Comment:
Not sure about naming for `Action<NSError> ItemLoadingHandler` - I went on the param naming of the binding directly above.

The documentation for `preparePreviewOfFileAtURL` says:

```
+ @param handler The completion handler should be called whenever the view is ready to be displayed. 
A loading spinner will be shown until the handler is called.
+ It can be called asynchronously after the method has returned.
```

Reply:
yeah, copy/paste is not always safe from past errors ;-)